README.md

```diff
@@ -1,6 +1,8 @@
 ## coreJSON Library
 
-A parser strictly enforcing the ECMA-404 JSON standard, suitable for microcontrollers
+This repository contains the coreJSON library, a parser that strictly enforces the ECMA-404 JSON standard and is suitable for microcontrollers.  It is distributed under the [MIT License](LICENSE)
```

Comment:
This repository contains the coreJSON library, a parser that follows the ECMA-404 JSON standard and is suitable for low memory footprint embedded devices. The coreJSON library is distributed under the [MIT open souce license](LICENSE)